content/02.body.md

```diff
@@ -12,40 +12,39 @@ Because different nodes can have very different degrees, real networks have high
+This evidence suggests that many poorly studied genes have similar numbers of interactions as those scientists have preferentially examined and that these edges are missed due to inspection bias.
+For networks with strong inspection bias, reliance on degree can lead to predictions that have a good metrics when assessed by cross validation but little ability to generalize.
 
 Another reason why a method's reliance on degree can be unfavorable is that degree imbalance can lead to prediction nonspecificity.
 Nonspecific predictions are made on the basis of generic characteristics rather than the specific connectivity information contained in a network.
 For example, Gillis et al. [@doi:10.1371/journal.pone.0017258] examined the concept of prediction specificity in the context of gene function prediction and found that many predictions appear to rely primarily on multifunctionality and could be "potentially misleading with respect to causality."
 Real networks have a variety of degree distributions (Figure {@fig:hetionet}), and they commonly exhibit degree imbalance [@doi:10.1371/journal.pone.0017645; @doi:10.1007/978-1-61779-361-5_13; @doi:10.1038/s41467-019-08746-5].
 Degree imbalance leads high-degree nodes to dominate in the predictions made by degree-associated methods [@doi:10.1093/bioinformatics/btv215], which are effective predictors of connections in some biological networks [@doi:10.1186/1752-0509-2-11].
-
 Consequently, degree-based predictions are more likely nonspecific, meaning the same set of predictions performs well for different tasks.
-However, depending on the prediction task, edge predictions between very high degree nodes may be undesired, uninsightful, or nonspecific.
-Model evaluation is challenging in this context, as nonspecific or trivial predictions need not be incorrect, even if they are not the desired outputs of the predictive model.
-For example, predicting that the highest degree node in a network shares edges with the remaining nodes to which it is not connected will often lead to many correct predictions, despite the predictions being generic to all nodes in the network.
-Degree-based features should often be included in the interpretation of predictions to disentangle desired from non-desired effects and to effectively evaluate and compare predictive models.
+
+Depending on the prediction task, edge predictions between very high degree nodes may be undesired, uninsightful, or nonspecific.
+Model evaluation is challenging in this context: nonspecific or trivial predictions can be correct, even if they are not the desired outputs of the predictive model, and dominate performance evaluations.
```

Comment:
I like the rewording of this paragraph a lot. What if we switch the two descriptions in this sentence?

```suggestion
Model evaluation is challenging in this context: nonspecific or trivial predictions can dominate performance evaluations and may actually be correct, even if they are not the desired outputs of the predictive model.
```